lints/lint_old_root_ca_rsa_mod_less_than_2048_bits.go

```diff
@@ -26,7 +26,11 @@ func (l *rootCaModSize) CheckApplies(c *x509.Certificate) bool {
 }
 
 func (l *rootCaModSize) RunTest(c *x509.Certificate) (ResultStruct, error) {
-	mod := c.PublicKey.(*rsa.PublicKey).N
+	key, found := c.PublicKey.(*rsa.PublicKey)
```

Comment:
If that is the case, then that lint is not correctly defined. Nowhere in `lints/lint_rsa_mod_less_than_2048_bits.go`, do you check for the validity period.